cdap-ui/app/cdap/components/Replicator/Create/Content/SelectTables/ManualSelectTable/ColumnsMultiRow/index.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as React from 'react';
+import AbstractMultiRowWidget, {
+  IMultiRowProps,
+} from 'components/AbstractWidget/AbstractMultiRowWidget';
+import ColumnRow from 'components/Replicator/Create/Content/SelectTables/ManualSelectTable/ColumnsMultiRow/ColumnRow';
+
+interface IColumnsMultiRowProps extends IMultiRowProps<{}> {
+  value: any;
+}
+
+class ColumnsMultiRow extends AbstractMultiRowWidget<IColumnsMultiRowProps> {
+  public componentWillReceiveProps() {
+    // overwrite
+    // no-op
+  }
+
+  public deconstructValues = (props) => {
+    if (!props.value || props.value.size === 0) {
```

Comment:
why do we use props.value.size here instead of length?

Reply:
good catch. Left over from when I was using immutableJS here